### ISSUE TITLE
Export to HTML fails

### ISSUE THREAD

**Issue by johnrico2000** [bladecoder/bladecoder-adventure-engine#5]
Hi Rafael,

today I tried to export a prototypical game made with your editor to HTML using "gradlw html:dist" and it failed probably due to some missing link to the bladecoder engine's source code. GWT needs access to the source code to able to compile things to JavaScript. Unfortunately I don't really know my way around gradle so I can't actually point out where to add the reference to the engine's source code in the gralde files. For my own game's export to HTML (http://hungryarchie.com) I was using the Eclipse IDE to export it to HTML.

I tried the export with your Goddess Robbery demo game, too, and it fails the same way (see screenshot). I really hope it's not a problem with my system, but currently I don't really think so.

Greetings from Germany,
Frank

![bladecoder-engine-html-export-fails](https://cloud.githubusercontent.com/assets/5022199/8473199/ce62ec38-20a7-11e5-9a78-718612ddf08d.jpg)

**Comment by bladecoder** [comment]
Hi Frank,

Currently only desktop, android and IOS are working properly. HTML needs some work. 

The main problem with HTML is that freetype library is not supported. We need to make the use of this library optional, but this require some time and until the v1.0 release it is low priority for me.

Regards,

Rafael.

**Comment by johnrico2000** [comment]
Congrats to the release of version 1.0. Impressive how consistently you are working on this project, Keep up the great work.

*cough* HTML export (maybe by adding fallback bitmap based font) *cough* ;-)

Greetings from Germany,
Frank

**Comment by bladecoder** [comment]
Yes, still a pending task... I don't see it working this year at least someone helps me...I have to release a game and then add LUA and Ink support...and maybe next...

**Comment by jack0088** [comment]
I fallowed the development process of this engine for a while now. You are doing great work and steady progress. Lua support sounds legit!
